tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +127,68 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_checks(self):
+        self.git_commit('Test commit')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+        self.git_commit('Revert "Test commit"\n\n'
+                        'Abcdef blablabla commit ' + str(commit_hash) + '\n\n'
```

Comment:
Test content should be self explanatory about its existence.
Please use proper sentence in place of
`Abcdef blablabla commit ` :P